shell/app/modules/project/pages/test-manage/case/header/export-file.tsx

```diff
@@ -27,6 +27,7 @@ const ExportFile = () => {
         i18n.t('application:The export task has been created, please check the progress in the record'),
         4,
       );
+      afterExport?.();
```

Comment:
if u add `?` means afterExport maybe undefined, but in the props definition, afterExport is required